src/components/BigCardListItem.js

```diff
@@ -0,0 +1,111 @@
+import PropTypes from 'prop-types';
+import React, { memo } from 'react';
+import { Platform, StyleSheet, View } from 'react-native';
+import { Card, Divider } from 'react-native-elements';
+
+import { colors, consts, normalize } from '../config';
+import { imageHeight, imageWidth } from '../helpers';
+
+import { Image } from './Image';
+import { CategoryText, HeadlineText, RegularText } from './Text';
+import { Touchable } from './Touchable';
+
+export const BigCardListItem = memo(({ navigation, horizontal, item }) => {
```

Comment:
following the same naming across everything related to the text list item with a card, i suggest `CardTextListItem`. so this naming runs through everywhere.

Reply:
I think I'll remove this component. I think the `CardListItem` component will be enough.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/920/commits/3a454a280b079b4a8d26d3f0edad7db5a24e7b56